sympy/sets/sets.py

```diff
@@ -1466,9 +1503,14 @@ class EmptySet(with_metaclass(Singleton, Set)):
 
     .. [1] https://en.wikipedia.org/wiki/Empty_set
     """
-    is_EmptySet = True
+    is_empty = True
     is_FiniteSet = True
 
+    @property
+    @deprecated(useinstead="is_empty", issue=16946, deprecated_since_version="1.5")
+    def is_EmptySet(self):
```

Comment:
Is this not inherited from `Set`?

Reply:
Unfortunately not, neither for `is_EmptySet = True` nor for a `def is_EmptySet(self)` property.

Reply:
Why not? I just tried this with a new class and it seems to work:
```julia
In [4]: class A(Set): 
   ...:     pass 
   ...:                                                                                                                           

In [5]: A().is_EmptySet                                                                                                           

In [6]: A().is_EmptySet == None                                                                                                   
Out[6]: True
```

Reply:
I'm not sure I'm following: The default for the deprecated `is_EmptySet` of the parent class is `None`, so it must be overridden to `True` using either `is_EmptySet` or `def is_EmptySet()`. But as far as I can tell the deprecation is not inherited so I have to repeat it, hence the `@deprecated def is_EmptySet()`.

Reply:
Oh sorry I see. You have to override it here to change the return value from None to True and then also need to add the deprecation. That makes sense.